cinema/models.py

```diff
@@ -41,11 +41,16 @@ class Movie(models.Model):
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(
+        upload_to="media/images/",
```

Comment:
nope, you had to write a function that does something like that;
 input: `image.jpg`
upload_to="upload/movies" --> 
output: `media/upload/movies/image-SOME-RANDOM_SYBMOLS.jpg`